packages/presentation/presentation_publishing/interfaces/has_force_refresh.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export interface HasForceRefresh {
+  forceRefresh: () => void;
```

Comment:
Should `forceRefresh` be part of `PublishesUnifiedSearch`

Reply:
Maybe we could do this as part of our next iteration on the `unifiedSearch` provider?